readthedocs/subscriptions/products.py

```diff
@@ -0,0 +1,120 @@
+"""
+Read the Docs products.
+
+This module contains the dataclasses that represent the products available.
+The available products are defined in the ``RTD_PRODUCTS`` setting.
+"""
+from dataclasses import dataclass
+
+from django.conf import settings
+
+from readthedocs.subscriptions.constants import FEATURE_TYPES
+
+
+@dataclass(slots=True)
+class RTDProductFeature:
+    # readthedocs.subscriptions.constants.TYPE_*
+    type: str
+    # Depending on the feature type, this number can represent a numeric limit, number of days, etc.
+    value: int = 0
+    # If this feature is unlimited for this product.
+    unlimited: bool = False
+    description: str = ""
+
+    def get_description(self):
+        if self.description:
+            return self.description
+        default_description = dict(FEATURE_TYPES).get(self.type)
+        if self.unlimited:
+            return f"{default_description} (unlimited)"
+        if self.value:
+            return f"{default_description} ({self.value})"
+        return default_description
+
+    def to_item(self):
+        """
+        Return a tuple with the feature type and the feature itself.
+
+        Useful to use it as a dictionary item.
+        """
+        return self.type, self
+
+
+@dataclass(slots=True)
+class RTDProduct:
```

Comment:
```suggestion
class Product:
```

I'd remove the `RTD` prefix from these classes since I think it's clear these are our products and features.

Reply:
I think the best way to differentiate them from the Stripe ones is using the syntax you are currently using, `djstripe.Product`, when required.

Reply:
I think this will make the code easier to read and follow.

Reply:
I'd like to keep the prefix, django shell_plus will import all models by default, so collision is going to be a problem there.